src/translations/dictionary.ts

```diff
@@ -74,6 +74,23 @@ const dictionary = {
       },
       long: {},
     },
+    relativeDayNames: (daysDifference: number) => {
+      switch (daysDifference) {
+        case -2:
+          return _('i forgårs', 'the day before yesterday', 'i forgårs');
+        case -1:
+          return _('i går', 'yesterday', 'i går');
+        case 0:
+          return _('i dag', 'today', 'i dag');
+        case 1:
+          return _('i morgen', 'tomorrow', 'i morgon');
+        case 2:
+          return _('i overmorgen', 'the day after tomorrow', 'i overmorgon');
+        default:
+          return _('', '', '');
```

Comment:
Shouldn't be better to show here like `in 3 days`, `in 5 days`, `20 days ago`?

Reply:
Good point :+1: